ui/src/wizard/AzureWizard.ts

```diff
@@ -20,14 +20,18 @@ export class AzureWizard<T> {
         this._wizardContext = wizardContext;
     }
 
-    public async prompt(actionContext: IActionContext, ui: IAzureUserInput): Promise<T> {
+    public async prompt(actionContext: IActionContext, ui: IAzureUserInput, parentWizard?: AzureWizard<T>): Promise<T> {
         for (const step of this._promptSteps) {
+            if (parentWizard && parentWizard._promptSteps.some((parentStep: AzureWizardPromptStep<T>) => parentStep.constructor.name === step.constructor.name)) {
```

Comment:
So this is to remove multiple instances of the same step?  Is it possible for this to be intended?  E.g. two location steps, one for a resource group and one for something else?

Reply:
Yes that's the point. With the current UI, I think we should encourage the least amount of prompts possible. Also two of the same steps won't work today with how the wizardContext is setup (Since it's a single `wizardContext` object being passed around, the second prompt would just overwrite the value from the first prompt).

I'm fine re-visiting this behavior once VS Code has better wizard support, but I think this is by-design for now.

Reply:
Make sense.